tealium/__init__.py

```diff
@@ -105,15 +126,16 @@ def enable(self):
             statusResult = False
         return statusResult
 
-    def trackEvent(self, title, eventType=None, data=None, tealiumCallback=None):
+    def trackEvent(self,  **arguments):
+
```

Comment:
the `**` argument is the ellipsis operator `...` in other languages. This seems to contradict what the docstring says. Is there a purpose for it?